new_course/1_programming_fundementals/debugging.md

```diff
@@ -128,13 +128,15 @@ def double_words_in_phrase(string)
   string_array.map { |word| word * 2}
   string_array.join(' ')
 end
+
+double_words_in_phrase("This is a test")
 ~~~
 
 When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`.
 
-Using Pry to debug achieves the same outcome as `puts` debugging: it allows you to confirm the assumptions you have about particular parts of your code. If your code is complex, Pry will probably allow you to debug quicker thanks to it's interactive, runtime environment. In such scenarios, Pry will be easier to interact with, rather than adding many `puts` statements everywhere and re-running your code again each time.
+Using Pry to debug achieves the same outcome as `puts` debugging: it allows you to confirm the assumptions you have about particular parts of your code. If your code is complex, Pry will probably allow you to debug quicker thanks to its interactive, runtime environment. Using Pry allows you to check your code while getting instant feedback. Whereas using `puts` debugging requires you to add many `puts` statements and re-run your code everytime you want to check for something new. Plus, with `puts` debugging you have to remember to go back and remove all those `puts` statements from you code.   
 
-There is a lot, lot more you can do with Pry which is out of the scope of this lesson. Check out the assignments and extra resources to find out where you can learn more about this useful gem.
+There is a lot, a lot more you can do with Pry which is out of the scope of this lesson. Check out the assignments and extra resources to find out where you can learn more about this useful gem.
```

Comment:
I thought you were trying to say... "There is a lot <pause> a lot more you can do with..." but looking back I think the original statement was saying something similar to: "There are lots lots more you can do with...".